Wiki/FAQ.md

```diff
@@ -0,0 +1,709 @@
+solo-stake, go through the following:
+
+If you have just begun your BOINC journey it is quite possible your work
+units haven't been verified by the necessary servers. Don't Panic,
+Gridcoin is a game of patience. Just relax, take a day or two, and see
+if the problem persists. If so, proceed to the following steps.
+
+To ensure your wallet is fully unlocked, lock your wallet with a
+passphrase (make it LONG, a password manager like KeePass is
+recommended) and then unlock it with "for staking only" UNticked.
+
+Next, go to Help -\> Debug Window, go to the Console tab and type
+`advertisebeacon` without the quotes.
+
+This will (if successful) send a small transaction at the cost of
+-0.00011 GRC. This should add your CPID with the next superblock.
+
+To see if your CPID is included, go to the Console and type `beaconreport`. If your CPID is present, sync the Neural Network by
+typing `syncdpor2` in the Console. If successful, the output
+should be '"syncing": 1'.
+
+If your CPID is not included, wait for the next superblock. In case your
+CPID isn't included after a few superblocks, try resending your beacon
+(`advertisebeacon`).
+
+Lock your wallet again, but this time unlock it with "for staking only"
+ticked.
+
+## How long does it take to start receiving Research (mining) credit?
+
+This depends on several factors. You must have some coins to begin with,
+some have started with as few as 100 coins, but the general recommendation now is around 5000. You must send a "beacon",
+this is how your information is populated into the Neural Network which
+controls payments, see [Mining setup](Mining-Setup "wikilink"). You must
+have over 100 RAC on at least 1 whitelisted BOINC project before you can
+send a beacon to receive Research credit. After your CPID is in the
+Neural Network, all research credits you earn will accrue in a "savings
+account" for a rolling 14 day period. You have up to 2 full weeks to
+stake a block without losing any research credit
+earned.
+
+## Is it true that someone would not lose reward if blocks are created on a fork?
+
+YES! If you find a block on a fork your "owed" balance will be reduced
+on that fork but it won't be reduced on the main block chain because
+that block wasn't really found by you on that fork. So once you sync
+back up with the main chain you will see the block you found change to
+an orphan (with the ?) and the balance in your research savings account
+will increase by that amount so that when you find a block on the main
+chain you will be paid for the research you are owed. This is what makes
+this method so much more fair, than the old PoW system. With
+Proof-of-Research as long as you find enough blocks in a 2 week window
+to be paid your full reward within the payout rules you will receive all
+the rewards you earned in the same period.
+
+## What should I put in my config-file for Gridcoin-Research?
+
+Please find gridcoinresearch.conf in your

```

Comment:
This can probably just say link to the Config-File page instead of explaining this the Config-File page is more likely to be up to date